scripts/smoke-test/commands.sh

```diff
@@ -0,0 +1,7 @@
+# Note this is not a real shell-script, it's parsed by `smoke-test/__main__.py` and executed
```

Comment:
We probably _could_ just have a list of shell scripts here and actually run them with a shell, but just doing something that fulfills the current needs right now.

Reply:
Can we just inline this as a string into `smoke-test/__main__.py`? It feels a little safer than reading and executing this file.

Reply:
I wanted syntax highlighting, but... okay :)

Reply:
I did this but... idk I actually do care about syntax highlighting so there can be comments. I don't want to clutter `__main__.py` as we add more commands and I don't want to have to deal with Python syntax.

Reply:
I think I'm going to do what I said above and just invoke separate shell scripts.

Reply:
Going back because `sh` isn't available on aarch64. We could switch in the future once it is.